docs/tutorials/new-feature-tutorial.md

```diff
@@ -171,30 +171,35 @@ guide to adding or updating documentation for an API endpoint.
 
 ## Example feature
 
-This example describes the process of adding a new setting to Zulip: a
-flag that allows an admin to require topics on stream messages (the default
-behavior is that topics can have no subject). This flag is an
-actual Zulip feature. You can review [the original commit](https://github.com/zulip/zulip/pull/5660/commits/aeeb81d3ff0e0cc201e891cec07e1d2cd0a2060d)
-in the Zulip repo. (This commit displays the work of setting up a checkbox
-for the feature on the admin settings page, communicating and saving updates
+This example describes the process of adding a new setting to Zulip.
+The implementation of this setting is similar to that of
+an actual Zulip feature. You can review [the original commit for this feature](https://github.com/zulip/zulip/pull/5660/commits/aeeb81d3ff0e0cc201e891cec07e1d2cd0a2060d)
+in the Zulip repo. This commit displays the work of setting up a checkbox
+for a feature on the admin settings page, communicating and saving updates
 to the setting to the database, and updating the state of the application
 after the setting is updated. For the code that accomplishes the underlying
-task of requiring messages to have a topic, you can [view this commit](https://github.com/zulip/zulip/commit/90e2f5053f5958b44ea9b2362cadcb076deaa975).)
+task of requiring messages to have a topic, you can [view this commit](https://github.com/zulip/zulip/commit/90e2f5053f5958b44ea9b2362cadcb076deaa975).
+
+You can view this feature rendered in the browser if you have the
+Zulip server running. It's the "Require topics in stream messages"
```

Comment:
"Zulip server" is probably confusing here. I think we should specifically refer to (and maybe link to the documentation for) the development environment.

Also, let's bold the setting title instead of using quotes: "It's the **Require topics in stream messages** checkbox ..."

Reply:
would this be the correct wording: "...if you are in the development environment." ?

Reply:
or perhaps "...if you are in the Zulip development environment."

Reply:
I bolded the settting title and went ahead with "Zulip development environment." for now, but I'll be happy to change it if there's a better way to word it :)